app/models/miq_queue.rb

```diff
@@ -48,6 +48,9 @@ def self.messaging_client(client_ref)
       # internally the client will track the state of the connection and re-open it,
       # once it's available again - at least thats true for a stomp connection
       ManageIQ::Messaging::Client.open(messaging_client_options.merge(:client_ref => client_ref))
+    rescue => err
+      _log.warn("Failed to open messaging client: #{err}")
+      nil
```

Comment:
This will cause it to constantly try to connect...will that be a problem, particularly if there are connection timeouts?

Reply:
Yeah, so I wanted to make sure that a hiccup connecting to kafka wouldn't cause issues processing events and metrics the normal way

We could keep a timestamp and only retry after a certain threshold?

Reply:
do we want some sort of retry mechanism?
Do we need a mechanism that resets a connection?

connection pools are hard